C-Plus-Plus/dp/Floyd_warshall.cpp

```diff
@@ -0,0 +1,106 @@
+#include<bits/stdc++.h>
+using namespace std;
+void floyd_waeshall(int arr[][5])
```

Comment:
@yashaswibiyahut Update `floyd_waeshall` to `floyd_warshall`.

Reply:
done @iamrajiv Kindly review it